manual/manual/tutorials/coreexamples.etex

```diff
@@ -677,6 +677,93 @@ print_expr e; print_newline ();;
 print_expr (deriv e "x"); print_newline ();;
 \end{caml_example}
 
+\section{Printf formats}
+
+There is a "printf" function in
+\href{https://caml.inria.fr/pub/docs/manual-ocaml/libref/Printf.html}{Printf}
+module (see chapter~\ref{c:moduleexamples}) that allows you to make formatted
+output more concisely.
+It follows the behavior of the "printf" function from the C standard library.
+The "printf" function takes a format string that describes the output values and
+their format, and arguments for output that correspond to the formatting
+directives in the format string.
```

Comment:
I have one last comment before approving and merging: I fear that the above sentence

> The "printf" function takes a format string that describes the output values ... in the format string.

 does not not work. Reading the sentence, I am left with four questions
- what are those `output values`?
- what are their format ?
- what is the difference between `output values` and `argument for output`?
- what are formatting directives?

On the first two questions, I think that you are trying to describe the specifiers, but then the proposition

> a format string that describes the output values and their format

is problematic because the format string is more than a collection of specifiers: `%a%a` ought to be different from `"%a +%a"`. Similarly, the last two questions arises because  the connection between the specifiers and the arguments is muddled by the apparition of a new term `formatting directives` which was never mentioned before.

 I think you meant something like

> The first argument of this "printf" function is a format string that describes the desired output as a text interspersed with specifiers (for instance "%d", "%f", \ldots) . The following arguments are then used to fill the holes left by the specifiers inside the format string:

Like always, feel free to reformulate the above text or even discard it, but keep in mind that this is an introduction, new terms should be as sparse as possible.

Reply:
In the last commit I tried to simplify as much as possible